src/random.string.generator.ts

```diff
@@ -14,7 +14,7 @@ export const ASCII = '!"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXY
  * @param length Desired length of the generated string. Defaults to 10.
  * @param charset Desired character set to select characters from at random. Default to Alphanumeric (a-zA-Z0-9).
  */
-export const randomString = (length: number = 10, charset: string = ALPHANUMERIC) => {
+export const randomString = (length = 10, charset: string = ALPHANUMERIC) => {
```

Comment:
Im surprised the linter gets upset about the number type declaration but not the string type

Reply:
That's a good point. Maybe it only does this for literals? I wonder if it would do the same thing if it were `length: number = MyEnum.TEN` or something like that.

Reply:
Yeah, this looks weird to me having a mix of typed and untyped variables... but whatever! We can tweak the rules later.